### DISCUSSION TITLE
Appwrite imange 1.3.7 not working on docker desktop win11

### DISCUSSION THREAD

**Discussion by eylonheiman** [appwrite/appwrite#5655] [General]
Hi
Using the command:
_docker run -it --rm \
    --volume /var/run/docker.sock:/var/run/docker.sock \
    --volume "$(pwd)"/appwrite:/usr/src/code/appwrite:rw \
    --entrypoint="install" \
    appwrite/appwrite:1.3.7_

console log output:

_Worker 1 started successfully
Worker 2 started successfully
Worker 4 started successfully
Worker 3 started successfully
Worker 5 started successfully
Worker 6 started successfully
Worker 7 started successfully
Worker 8 started successfully
Database not ready. Retrying connection (1)...
Database not ready. Retrying connection (2)...
Database not ready. Retrying connection (3)...
Database not ready. Retrying connection (4)...
Database not ready. Retrying connection (5)...
Database not ready. Retrying connection (6)...
Database not ready. Retrying connection (7)...
Database not ready. Retrying connection (8)...
Database not ready. Retrying connection (9)...
Database not ready. Retrying connection (10)...

Fatal error: Uncaught Exception: Failed to connect to database: SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo failed: Try again in /usr/src/code/app/http.php:77
Stack trace:
#0 {main}
  thrown in /usr/src/code/app/http.php on line 77
[2023-06-05 07:15:58 #1.8]      ERROR   php_swoole_server_rshutdown() (ERRNO 503): Fatal error: Uncaught Exception: Failed to connect to database: SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo failed: Try again in /usr/src/code/app/http.php:77
Stack trace:
#0 {main}
  thrown in /usr/src/code/app/http.php on line 77_

After about a minute, the container exited.
What did I do wrong? 

Thanks

**Comment by joeyouss** [comment]
Hi
What does `docker ps -a` show?

**Reply by joeyouss** [reply]
Also, please see this once: https://github.com/appwrite/appwrite/issues/3676

**Comment by joeyouss** [comment]
Hi @eylonheiman , did it work?
